src/lib_wasm.rs

```diff
@@ -4,11 +4,16 @@
 **/
 extern crate base64;
 
+use std::collections::HashMap;
+
 use crate::{
-    rewriter::{print_js, rewrite_js},
+    rewriter::{print_js, rewrite_js, Config},
+    telemetry::{Telemetry, TelemetryVerbosity},
```

Comment:
Why do you need `Telemetry` here but not in `src/lib_napi.rs`?

Reply:
The same as above